### PR TITLE
ios: translations

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: apps/ios/SimpleX Localizations/de.xcloc/Localized Contents/de.xliff [hunk 1]
```diff
@@ -3504,7 +3544,8 @@ SimpleX-Server können Ihr Profil nicht einsehen.</target>
       <trans-unit id="off" xml:space="preserve">
         <source>off</source>
         <target>***off</target>
-        <note>group pref value</note>
+        <note>enabled status
+   group pref value</note>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/de.xcloc/Localized Contents/de.xliff:3548]: accidental new line?
```suggestion
        <note>enabled status group pref value</note>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 2]
```diff
@@ -1683,16 +1673,16 @@ We will be adding server redundancy to prevent lost messages.</source>
...
-        <note>No comment provided by engineer.</note>
-      </trans-unit>
       <trans-unit id="Make a private connection" xml:space="preserve">
         <source>Make a private connection</source>
         <target>Добавьте контакт</target>
         <note>No comment provided by engineer.</note>
       </trans-unit>
+      <trans-unit id="Make sure SMP server addresses are in correct format, line separated and are not duplicated (%@)." xml:space="preserve">
+        <source>Make sure SMP server addresses are in correct format, line separated and are not duplicated (%@).</source>
+        <target>Пожалуйста, проверьте, что адреса SMP серверов имеют правильный формат, каждый адрес на отдельной строке и не повторяется (%@).</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:1683]: this seems like an outdated error message.. have it missed an alert somewhere?

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 3]
```diff
@@ -3509,7 +3544,8 @@ SimpleX серверы не могут получить доступ к ваше
       <trans-unit id="off" xml:space="preserve">
         <source>off</source>
         <target>нет</target>
-        <note>group pref value</note>
+        <note>enabled status
+   group pref value</note>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:3548]: same
```suggestion
        <note>enabled status group pref value</note>
```

File: apps/ios/SimpleX Localizations/en.xcloc/Localized Contents/en.xliff [hunk 4]
```diff
@@ -3504,7 +3544,8 @@ SimpleX servers cannot see your profile.</target>
       <trans-unit id="off" xml:space="preserve">
         <source>off</source>
         <target>off</target>
-        <note>group pref value</note>
+        <note>enabled status
+   group pref value</note>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/en.xcloc/Localized Contents/en.xliff:3548]: same
```suggestion
        <note>enabled status group pref value</note>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff (hunk 2 above)
Comment (epoberezkin) [reply apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:1683]: no, there is an alert, just in case

**Pull request merged**